connector/server/explore/explore_query_logic.go

```diff
@@ -222,7 +270,8 @@ func (q *ExploreQuery) getI2b2PsmQueryTerms(taggedQueryTerms map[string]string)
 				}
 
 			} else {
-				itemKey = *item.QueryTerm
+				itemKey = `\` + strings.ReplaceAll(*item.QueryTerm, "/", `\`)
```

Comment:
If you look at the if block of this part, the sensitive tag key is written using the backslashes: \\SENSITIVE_TAGGED\medco\tagged\` + tag + `\`. The quick fix of the else statement simply applies the same format.

Do you suggest the client should send paths written with backslashes instead of slashes in the first place ?

Reply:
OK you are right I got confused here.